### DISCUSSION TITLE
Connect to a remote database?

### DISCUSSION THREAD

**Discussion by fengyaocheng** [akiver/cs-demo-manager#708] [Q&A] [answered]
Can I connect to a remote database on windows.exe? For example, when connecting to the data I deployed on Linux, it now prompts me:

PSQL binary not found

This usually means that PostgreSQL is not installed on your system, or the path to the psql executable file is not in the PATH environment variable.

**Comment by akiver** [comment, marked as answer]
Yes you can , I added a doc about this [here](https://cs-demo-manager.com/docs/guides/database#using-a-remote-database).

You have to install **only the psql CLI** on your local machine.
